data/multiregion.json

```diff
@@ -2,14 +2,14 @@
   "dataset_type": "multiregion",
   "path": "data/multiregion.csv",
   "data_git_info": {
-    "sha": "623fa8940f475d3eb6dc0b3c4c9cdb6c7f4979a9",
-    "branch": "master",
+    "sha": "d6c4bfd5f098ddbbf63cf0a15fd22fbbe9114eb7",
+    "branch": "mikelehen/drop-county-hhs-data",
```

Comment:
I won't merge all of these data files.  I'll just merge the code and then run the github action to update combined datasets.